libstorage/SQLStorage.cpp

```diff
@@ -107,48 +114,87 @@ Entries::Ptr SQLStorage::select(h256 hash, int64_t num, TableInfo::Ptr tableInfo
                 -1, "Remote database return error:" + boost::lexical_cast<std::string>(code));
         }
 
-        std::vector<std::string> columns;
-        for (Json::ArrayIndex i = 0; i < responseJson["result"]["columns"].size(); ++i)
-        {
-            std::string fieldName = responseJson["result"]["columns"].get(i, "").asString();
-            columns.push_back(fieldName);
-        }
-
         Entries::Ptr entries = std::make_shared<Entries>();
-        for (Json::ArrayIndex i = 0; i < responseJson["result"]["data"].size(); ++i)
+        if (g_BCOSConfig.version() <= RC3_VERSION)
         {
-            Json::Value line = responseJson["result"]["data"].get(i, "");
-            Entry::Ptr entry = std::make_shared<Entry>();
+            std::vector<std::string> columns;
+            for (Json::ArrayIndex i = 0; i < responseJson["result"]["columns"].size(); ++i)
+            {
+                std::string fieldName = responseJson["result"]["columns"].get(i, "").asString();
+                columns.push_back(fieldName);
+            }
 
-            for (Json::ArrayIndex j = 0; j < line.size(); ++j)
+            for (Json::ArrayIndex i = 0; i < responseJson["result"]["data"].size(); ++i)
             {
-                std::string fieldValue = line.get(j, "").asString();
+                Json::Value line = responseJson["result"]["data"].get(i, "");
+                Entry::Ptr entry = std::make_shared<Entry>();
 
-                if (columns[j] == ID_FIELD)
+                for (Json::ArrayIndex j = 0; j < line.size(); ++j)
                 {
-                    entry->setID(fieldValue);
+                    std::string fieldValue = line.get(j, "").asString();
+
+                    if (columns[j] == ID_FIELD)
```

Comment:
https://github.com/FISCO-BCOS/FISCO-BCOS/blob/623ba91f99ae7fab08971618378a74094dbd634d/libstorage/RocksDBStorage.cpp#L79-L85

can we do same as RocksDBStorage ?